specification/web/resource-manager/Microsoft.DomainRegistration/stable/2024-11-01/Domains.json

```diff
@@ -0,0 +1,1616 @@
+      "type": "object",
+      "properties": {
+        "keywords": {
+          "description": "Keywords to be used for generating domain recommendations.",
+          "type": "string"
+        },
+        "maxDomainRecommendations": {
+          "format": "int32",
+          "description": "Maximum number of recommendations.",
+          "type": "integer"
+        }
+      }
+    },
+    "HostName": {
+      "description": "Details of a hostname derived from a domain.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Name of the hostname.",
+          "type": "string"
+        },
+        "siteNames": {
+          "description": "List of apps the hostname is assigned to. This list will have more than one app only if the hostname is pointing to a Traffic Manager.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "azureResourceName": {
+          "description": "Name of the Azure resource the hostname is assigned to. If it is assigned to a Traffic Manager then it will be the Traffic Manager name otherwise it will be the app name.",
+          "type": "string"
+        },
+        "azureResourceType": {
+          "description": "Type of the Azure resource the hostname is assigned to.",
+          "enum": [
+            "Website",
+            "TrafficManager"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "AzureResourceType",
+            "modelAsString": false
+          }
+        },
+        "customHostNameDnsRecordType": {
+          "description": "Type of the DNS record.",
+          "enum": [
+            "CName",
+            "A"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "CustomHostNameDnsRecordType",
+            "modelAsString": false
+          }
+        },
+        "hostNameType": {
+          "description": "Type of the hostname.",
+          "enum": [
+            "Verified",
+            "Managed"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "HostNameType",
+            "modelAsString": false
+          }
+        }
+      }
+    },
+    "NameIdentifierCollection": {
+      "description": "Collection of domain name identifiers.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "../../../Microsoft.Web/stable/2024-11-01/CommonDefinitions.json#/definitions/NameIdentifier"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        },
+        "nextLink": {
+          "description": "Link to next page of resources.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    }
+  },
+  "parameters": {
+    "subscriptionIdParameter": {
```

Comment:
>subscriptionIdParameter [](http://example.com/codeflow?start=5&length=23)

Use the definition in common types instead.

Reply:
MUHAMMAD IHSAN ALI.
![image](https://github.com/user-attachments/assets/b9f17684-994c-47bc-810d-44be8aa7fea8).
![image](https://github.com/user-attachments/assets/c39f4ec9-0f5d-48a0-9a45-b7f79a01196e).
![image](https://github.com/user-attachments/assets/1b38aba0-4c22-4873-b515-68ce7625f6e2).
![image](https://github.com/user-attachments/assets/1c3baaa1-53aa-419c-8c8b-31c7b5634d74).
![image](https://github.com/user-attachments/assets/ac6d335a-3349-452b-b028-60a02ae25d37).
![image](https://github.com/user-attachments/assets/74a6ea71-d904-430a-8bd9-915bde1a315c).
![image](https://github.com/user-attachments/assets/d7c64b44-8f52-4cea-8cae-90bec186c8f0).
![image](https://github.com/user-attachments/assets/6a311668-6172-440d-82fd-ff99bc41ee4d).
![image](https://github.com/user-attachments/assets/d55a021d-9deb-4f37-a806-a46874098af4).
![image](https://github.com/user-attachments/assets/00f2740d-cd2a-4d5a-b088-a4de7c2ef23e).
![image](https://github.com/user-attachments/assets/8578f0ce-7536-4828-bb4b-038ccb88adc9).

Reply:
MUHAMMAD IHSAN ALI.INDEX.